docs/en/upgrade.md

```diff
@@ -91,17 +91,36 @@ The solution automatically and asynchronously upgrades the views and materialize
 6. Execute the following SQL to check the stored procedure execution log; ensure there are no errors. If there are any interruptions, timeouts, or other errors, you can re-execute step 4 to continue the data migration.
 
     ```sql
-    -- please replace `<app-id>` with your actual app id
-    SELECT * FROM "<app-id>"."clickstream_log" WHERE log_name = 'sp_migrate_event_to_v2' ORDER BY log_date DESC;
-    SELECT * FROM "<app-id>"."clickstream_log" WHERE log_name = 'sp_migrate_user_to_v2' ORDER BY log_date DESC;
-    SELECT * FROM "<app-id>"."clickstream_log" WHERE log_name = 'sp_migrate_item_to_v2' ORDER BY log_date DESC;
-    SELECT * FROM "<app-id>"."clickstream_log" WHERE log_name = 'sp_migrate_session_to_v2' ORDER BY log_date DESC;
-    SELECT * FROM "<app-id>"."clickstream_log" WHERE log_name = 'sp_migrate_data_to_v2' ORDER BY log_date DESC;
+    -- please replace <app-id> with your actual app id
+    SELECT * FROM "<app-id>"."clickstream_log_v2" WHERE log_name = 'sp_migrate_event_to_v2' ORDER BY log_date DESC;
+    SELECT * FROM "<app-id>"."clickstream_log_v2" WHERE log_name = 'sp_migrate_user_to_v2' ORDER BY log_date DESC;
+    SELECT * FROM "<app-id>"."clickstream_log_v2" WHERE log_name = 'sp_migrate_item_to_v2' ORDER BY log_date DESC;
+    SELECT * FROM "<app-id>"."clickstream_log_v2" WHERE log_name = 'sp_migrate_session_to_v2' ORDER BY log_date DESC;
+    SELECT * FROM "<app-id>"."clickstream_log_v2" WHERE log_name = 'sp_migrate_data_to_v2' ORDER BY log_date DESC;
     ```
 
-7. Follow [this guide][faq-recalculate-data] to calculate metrics for the new preset dashboard based on the migrated data.
+7. populate the event data to `clickstream_event_base_view` table.
+
+    ```sql
+    -- please replace <app-id> with your actual app id
+    -- update the day range(180 days in below example) based on your needs
+    CALL "<app-id>".clickstream_event_base_view_sp(NULL, NULL, 24*180);
+    ```
+
+    !!! info "Note"
+
+        If new events come before completing this step or the volume of migrated data is large, it is recommended to refresh the `clickstream_event_base_view` in batches. That is, based on the occurrence time of the events, call the following stored procedure multiple times to refresh the data in batches(needs to be in order from old to new event dates).
```

Comment:
It is recommended to refresh the clickstream_event_base_view in batches, especially in the following scenarios:
- When there are new event load jobs coming in before the migration job completes.
- When the volume of migrated data is large (e.g., 100 millions in a batch).

Noe that refreshing the data in batches needs to be done based on the event timestamp. Call the following stored procedure multiple times, in order from old to new event timestamps.